### ISSUE TITLE
I cannot get more than 4 devices to work.

### ISSUE THREAD

**Issue by chrisbussey** [hardillb/node-red-alexa-home-skill-web#51]
I have 10 Sonoff WiFi switches that are controlled via MQTT.  The inputs to the 10 MQTT nodes are from 10 alexa home skills nodes named Switch1.......Switch10.  The first 4 work perfectly but from 5 upwards Alexa tells me that she cannot find them even though she discovered them.
I have tried renaming nos. 5-8 as Light 1......... Light4 and renamed 9 and 10 as Motor 1 and 2 but Alexa only ever controls the first 4.
Any good ideas would be welcome.

**Comment by hardillb** [comment]
There should be no limits on the number of devices you can create with the skill. I have 8 that I use but they are a mix of lights, sockets and thermostats.

The appliance type setting should have no effect, it just makes using devices as part of room groups easier.

Can you turn the devices on/off via the Alexa mobile app?

Also what exactly is the error message that Alexa says when you try and use the devices?

Do you get any output from the node bound to that device in Node-RED? Are you using the auto acknowledge setting or responding manually?

And you are 100% sure you re-ran discovery after changing the names in the web app?

**Comment by chrisbussey** [comment]
Hi Ben, Thanks for the speedy reply.
When I "asked" (verbally) Alexa to turn on a device, Switch 5 for example, she came back saying she had multiple devices with that name even though there was only one in the alexa app and one in the home skills list.
This morning I removed all devices in the home skills list and all devices in the alexa app, I then did a new device discovery (of course she found nothing).  I then added Lights 1 through 10 in the alexa home skills list and did a rediscovery, she found 10.
I created a new very simple flow and all appears to be working.
I guess in my playing around I got some database screwed up and clearing everything out and starting again has sorted it.
Thanks very much.

Chris


![screen shot 2018-02-25 at 10 09 23](https://user-images.githubusercontent.com/12814083/36640329-206a3276-1a14-11e8-892b-d0936b471f05.png)
